Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -73,6 +73,7 @@ def __init__(self, self_deployed, tenant_id, auth_and_token_url, enc_key, app_na
         self._folder_to_fetch = folder_to_fetch
         self._first_fetch_interval = first_fetch_interval
         self._emails_fetch_limit = emails_fetch_limit
+        self._timeout = timeout
```

Comment:
This needs to be added to `MicrosoftClient` class and not this one such that it will be used automatically without needing to pass it from each function.

Reply:
done